learn-ts/src/02/problem-set.ts

```
@@ -0,0 +1,256 @@
+  let complexStructure: {
+    id: number;
+    value: string;
+    attributes: { weight: number; color: string };
+  }[] = [
+    { id: 1, value: "apple", attributes: { weight: 1.2, color: "green" } },
+    { id: 2, value: "banana", attributes: { weight: 0.5, color: "yellow" } },
+    { id: 3, value: "cherry", attributes: { weight: 0.2, color: "red" } },
+  ];
+  console.log(complexStructure);
+
+  let nestedData: { id: number; tags: Array<String>; name: string }[] = [
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)